app/src/main/java/io/pslab/fragment/PSLabPinLayoutFragment_v6.java

```diff
@@ -0,0 +1,272 @@
+package io.pslab.fragment;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.graphics.Matrix;
+import android.graphics.PointF;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AlertDialog;
+import androidx.core.content.res.ResourcesCompat;
+import androidx.fragment.app.Fragment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.pslab.R;
+import io.pslab.items.PinDetails_v6;
+
+public class PSLabPinLayoutFragment_v6 extends Fragment implements View.OnTouchListener {
+
+    private final List<PinDetails_v6> pinDetails_v6 = new ArrayList<>();
+
+    private final Matrix matrix = new Matrix();
+    private final Matrix savedMatrix = new Matrix();
+
+    public static boolean topside = true;
+
+    private static final int NONE = 0;
+    private static final int DRAG = 1;
+    private static final int ZOOM = 2;
+    private int mode = NONE;
+
+    private final PointF start = new PointF();
+    private final PointF mid = new PointF();
+    private float oldDist = 1f;
+
+    private ImageView colorMap;
+    private ImageView imgLayout;
+
+    public static PSLabPinLayoutFragment_v6 newInstance() {
+        return new PSLabPinLayoutFragment_v6();
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_pin_layout_v6, container, false);
+        imgLayout = view.findViewById(R.id.img_pslab_pin_layout_v6);
+        colorMap = view.findViewById(R.id.img_pslab_color_map_v6);
+        return view;
+    }
+
+    @SuppressLint("ClickableViewAccessibility")
+    @Override
+    public void onResume() {
+        super.onResume();
+        imgLayout.setImageDrawable(ResourcesCompat.getDrawable(getResources(),
+                topside ? R.drawable.pslab_v6_bottom : R.drawable.pslab_v6_top, null));
```

Comment:
Wrong check.
This causes the bottom layout to be displayed when `topside` is true, and top layout to be displayed when `topside` is false.
```suggestion
                topside ? R.drawable.pslab_v6_top : R.drawable.pslab_v6_bottom, null));
```